### PR TITLE
Infobox component

### PR DESCRIPTION
**Pull request by michaljach**
screen shot 2017-08-17 at 2 25 27 pm

### CODE REVIEW

File: src/app/app.component.html [hunk 1]
```diff
@@ -1,3 +1,12 @@
 <div class="container">
   <app-header></app-header>
+  <div class="main">
+    <app-info
+      name="Marcin Smola"
+      date="24.12.1990"
```
Comment (smolamarcin) [inline src/app/app.component.html]: 24.07.1994

**Comment by smolamarcin** [comment]
Everything is ok, but i think round photo will be better.

File: src/app/app.component.html (hunk 1 above)
Comment (michaljach) [inline src/app/app.component.html]: Fixed.

**Comment by michaljach** [comment]
Changed avatar to be rounded.

**Pull request merged**